<launch>

    <node name="portal_node" pkg="portal" type="portal_node.py" respawn="true" output="screen"/>
    <node name="rosserial_portal_node" pkg="rosserial_python" type="serial_node.py" args="/dev/arduino_portal"/>

</launch>
